sopel/irc.py

```diff
@@ -367,12 +367,18 @@ def error(self, trigger=None):
             except Exception as e:
                 stderr("Could not save full traceback!")
                 LOGGER.error("Could not save traceback from %s to file: %s", trigger.sender, str(e))
-
-            if trigger and trigger.sender is not None:
-                self.msg(trigger.sender, signature)
+            
+            # Errors will pass silently otherwise, not sure how to handle this
+            if trigger and self.config.core.message_logs and self.config.core.logging_channel is not None:
+                self.msg(self.config.logging_channel, signature)
+                if self.config.core.message_logs_sender:
+                    self.msg(trigger.sender,signature)
         except Exception as e:
-            if trigger and trigger.sender is not None:
-                self.msg(trigger.sender, "Got an error.")
+            if trigger:
+                if self.config.core.message_logs and self.config.core.logging_channel is not None:
+                    self.msg(self.config.core.logging_channel,"Got an error.")
```

Comment:
No need for this line, `LOGGER.error` is sent to the `logging_channel` just below and that's more verbose than this is.